helix-view/Cargo.toml

```diff
@@ -42,6 +43,7 @@ toml = "0.5"
 log = "~0.4"
 
 which = "4.2"
+alacritty_terminal = "0.17.0"
```

Comment:
Can you add this dependency only in the `helix-vte` crate, then re-export it via `pub use alacritty_terminal` there? That way it's not included several times

Reply:
I really don't think `helix-vte` should know anything about `alacritty_terminal`. `helix-vte` it is about process PTY or SSH.

Reply:
Hmm okay. How about splitting `helix-view/src/terminal.rs` etc into a `helix-terminal` crate? I'd like to keep the least amount of code possible in `helix-view` so it's more separated from the rest of `-view`

Reply:
Only concern is - the name (`helix-terminal`) is confusing with `helix-term`